app/views/home/check.html.erb

```diff
@@ -1,13 +1,13 @@
 <head>
-  <title>Flashcards</title>
+  <title>Flashcard Manager (DE/EN)</title>
   <link rel="stylesheet" href="application.css">
```

Comment:
@kennyfrc check out the layouts/application template, you have already included css and js files there. 
also, you don't need to repeat yourself in different files, just move the navigation menu to layouts/application and it will automatically appear on all pages